resources/schema.py

```diff
@@ -328,6 +348,62 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         return DeleteBerthTypeMutation()
 
 
+class CreateHarborMutation(graphene.ClientIDMutation):
+    class Input:
+        area = HarborAreaInput(required=True)
+        municipality = graphene.String()
+        image_file = graphene.String()
```

Comment:
uploading files through graphene is not that straightforward, we would need to use [this package](https://github.com/lmcgartland/graphene-file-upload) or some other that implements [this spec](https://github.com/jaydenseric/graphql-multipart-request-spec)

Reply:
if you'd like, you could skip it in this PR, and imeplement it later, we can create a ticket. would avoid extra complixity in this PR 🙂

Reply:
Opened [VEN-405](https://helsinkisolutionoffice.atlassian.net/browse/VEN-405)